### PR TITLE
[1.19] Fix sound wrappers

### PR DESCRIPTION
**Pull request by NoJokeFNA**
WrapperPlayServerNamedSoundEffect doesn't exist yet, so nothing can be fixed there

### CODE REVIEW

File: api/src/main/java/com/github/retrooper/packetevents/wrapper/play/server/WrapperPlayServerEntitySoundEffect.java [hunk 1]
```diff
@@ -50,6 +59,9 @@ public void read() {
         this.entityId = readVarInt();
         this.volume = readFloat();
         this.pitch = readFloat();
+        if (serverVersion.isNewerThanOrEquals(ServerVersion.V_1_19)) {
+            this.seed = readLong();
```
Comment (MWHunter) [inline api/src/main/java/com/github/retrooper/packetevents/wrapper/play/server/WrapperPlayServerEntitySoundEffect.java:63]: https://wiki.vg/Protocol#Entity_Sound_Effect

is protocol wiki wrong?  I don't see this

**Review by MWHunter — CHANGES_REQUESTED** [review]

File: api/src/main/java/com/github/retrooper/packetevents/wrapper/play/server/WrapperPlayServerEntitySoundEffect.java (hunk 1 above)
Comment (NoJokeFNA) [reply api/src/main/java/com/github/retrooper/packetevents/wrapper/play/server/WrapperPlayServerEntitySoundEffect.java:63]: No you are right, sleepy eyes

**Comment by retrooper** [comment]
For us it's been a "convention" to keep method names for IDs ending with getId() instead of getID()

**Comment by retrooper** [comment]
So your change would make it inconsistent.

**Comment by retrooper** [comment]
Seems like the OptionalLong is useless as you never make it empty, its either null or has a value?

**Comment by NoJokeFNA** [comment]
But why would you name your field *ID* and your Getter/Setter *Id*? This is very illogical and contradictory for me

**Comment by retrooper** [comment]
True, variable naming is contradictory, maybe we'll change that but lets keep the methods the same.

**Comment by retrooper** [comment]
I don't see the point of the OptionalLong tho. You're right its never null.

**Comment by NoJokeFNA** [comment]
Well, Optionals exist for one reason, to check if the value does exist.

**Comment by retrooper** [comment]
In your case it always exists and is never empty is my point.

**Comment by NoJokeFNA** [comment]
Changed it, you are right

**Pull request merged**